examples/complete-nlb/variables.tf

```diff
@@ -1,5 +1,6 @@
 variable "domain_name" {
   description = "The domain name for which the certificate should be issued"
   type        = string
-  default     = "terraform-aws-modules.modules.tf"
+  # default     = "terraform-aws-modules.modules.tf"
+  default = "sharedservices.clowd.haus"
```

Comment:
Maybe put it into `terraform.tfvars`? I'm unsure which approach is best for both of us to be able to run these examples without manual changes.